pkg/scm/gitlab/context.go

```diff
@@ -158,3 +164,47 @@ func (c *Context) HasExecutedActionGroup(group string) bool {
 
 	return ok
 }
+
+// GetCodeOwners returns the eligible code owners for the merge request
+//
+// This is based on the elibible approvers in the rules of the merge request approval
+// state api.
+func (c *Context) GetCodeOwners() []scm.Actor {
```

Comment:
would be better suited in a gitlab-specific instance of the new action instead of the gitlab-specific context imo, but can be here for now

Reply:
feel free to just cast `scm.EvalContext` to `gitlab.EvalContext` inside the `gitlab` pkg and remove this from the function from the public `scm.EvalContext` API :)

Reply:
updated